academy/0.0-B9lab-Blockchains/2_public.md

```diff
@@ -0,0 +1,178 @@
+
+All this may not seem important but it is. Blockchain technology was built with disruption in mind. It was envisaged as the antithesis of the central control of banks, governments, and incumbent holders of monopolies. This has an influence on the direction of the technology and remains a strong influence on its development.
+
+Given its anti-establishment roots, blockchain technology has been seized upon by other groups interested in circumventing government, law enforcement, or regulatory control, both on the radical right and left fringes, as well as among activists persecuted by their governments.
+
+</ExpansionPanel>
+
+## 2010s
+
+As soon as it became clear that the technology presents a very attractive base infrastructure for payments, other groups and organisations adapted the technology for their purposes or developed new approaches using the same basic principles but adapting them to more traditional use cases.
+
+In 2014 Vitalik Buterin started what is now known as the second wave of blockchain technology by publishing the paper underlying the Ethereum blockchain protocol, [A Next Generation Smart Contract & Decentralised Application Platform (Vitalik Buterin, Ethereum’s creator)](https://github.com/ethereum/wiki/wiki/White-Paper). Beyond a distributed ledger, the Ethereum white paper proposed the development of a distributed computing platform.
+
+<HighlightBox type="tip">
+
+Have a closer look at the [Ethereum white paper](https://github.com/ethereum/wiki/wiki/White-Paper).
+
+</HighlightBox>
+
+The Ethereum project raised roughly $20 million in one of the most successful crowdfunding campaigns up to that point. The first public network was up and running in 2015. Simultaneously, it spawned the first managed blockchain network approach when the company Eris forked a version of Ethereum and expanded it to implement a layer of permissions, as well as making it easier to deploy custom, access controlled networks.
+
+Late 2015 saw the establishment of the [Hyperledger Foundation](https://www.hyperledger.org/), an industry consortium with a focus on enterprise blockchain technology for managed networks and business applications.
+
+In 2017 the [Ethereum Enterprise Alliance](https://entethalliance.org/) was formed as an industry consortium to adapt Ethereum for enterprise use.
+
+This brief background introduces the vibrant community and fast-paced evolution of the space from which Cosmos would emerge. 
+
+Today we can see two broad trends in the development and adoption of blockchain technology: public blockchains and managed/private blockchains. Understanding this distinction will be important to your understanding of Cosmos. Cosmos is applicable to both use-cases and, importantly, enables seamless interoperability that has challenged predecessors. 
+
+Let us start by diving into public blockchains and their most known examples, Bitcoin and Ethereum.
+
+## Public Blockchains
```

Comment:
I would strongly recommend moving the content on public blockchains into the next section to focus this section on one aspect. Recommendation is implemented in review changes.